yt/visualization/plot_modifications.py

```diff
@@ -2050,9 +2078,9 @@ class ScaleCallback(PlotCallback):
     _type_name = "scale"
     _supported_geometries = ("cartesian", "spectral_cube", "force")
     def __init__(self, corner='lower_right', coeff=None, unit=None, pos=None,
-                 max_frac=0.16, min_frac=0.015, coord_system='axis',
-                 text_args=None, size_bar_args=None, draw_inset_box=False,
-                 inset_box_args=None):
+                 scale_text_format="{scale} {units}", max_frac=0.16, 
```

Comment:
Super minor, but maybe have scale_text_format at the end to preserve the previous kwarg ordering?